### DISCUSSION TITLE
Question regarding aligning of Scenes in Sentinel on GMTSAR

### DISCUSSION THREAD

**Discussion by shivamthescienceguy** [gmtsar/gmtsar#321] [General]
Hie,
I have a doubt regarding the aligning(coregistration) of scenes in Sentinel. As there is no align.in in file, it uses data.in file right?
So the format of data.in is 
SCENE1: EOF_of_SCENES1 
SCENE2: EOF_of_SCENES2 
SCENE3: EOF_of_SCENES3
...
...
![image](https://user-images.githubusercontent.com/88940973/167080625-38a6b2b9-fee9-4e64-97ff-098689a1d17e.png)

So where is the master is coregistered with other aligned images like its was in the case for ALOS data where we selected **Master, Supermaster and Aligned**
So how all the images are aligned with each other?

**Comment by Xiaohua-Eric-Xu** [comment]
Hi @shivamthescienceguy , Sentinel-1 uses geometric co-registration and all orbits are within a ~200-m tube so the concept of leap-frog alignment is not necessary. For more information related to how the data is handled, attached is our paper. Xu, X., Sandwell, D. T., Tymofyeyeva, E., González-Ortega, A., & Tong, X. (2017). Tectonic and anthropogenic deformation at the Cerro Prieto geothermal step-over revealed by Sentinel-1A InSAR. IEEE Transactions on Geoscience and Remote Sensing, 55(9), 5284-5292.
